app/src/main/java/com/example/nvdovin/weatherapp/domain/utils/design/WeatherCodesMap.java

```diff
@@ -9,8 +9,8 @@
 
 public class WeatherCodesMap {
     private Map<String, List<Integer>> map;
-    private int iconResIndex;
-    private int bckgResIndex;
+    private final int iconResIndex;
```

Comment:
there is no reason to assign this variable with value in constructor, in this case is better to use constant instead just final variable that is initialized in  constructor. the same for the variable below.